client/src/pages/SurveyDetail/index.jsx

```diff
@@ -239,10 +229,18 @@ const SurveyDetail = ({ match }) => {
     successfulClose,
   } = useSelector((state) => state.surveyDetailReducer);
 
+  const { employeeData } = useSelector((state) => state.employeeTableReducer);
+
   useEffect(() => {
     const { id } = match.params;
     dispatch({ type: 'RESET_EMPLOYEE_DATA' });
     dispatch({ type: 'RESET_SURVEY_DETAIL_STATE' });
+    const getEmployees = async () => {
+      const { data } = await axios.get(`/employees`);
+      dispatch({ type: 'SET_EMPLOYEE_DATA', payload: data });
```

Comment:
I think set_employee_data is my state for the table. Would it make sense to have your own state if this is being used for the slack modal, but set it in the same place after the get? it seems weird for the state to be in the table reducer